packages/server/src/api/controllers/row/index.ts

```diff
@@ -168,7 +183,15 @@ async function deleteRows(ctx: UserCtx<DeleteRowRequest>) {
   }
 
   for (let row of rows) {
-    ctx.eventEmitter && ctx.eventEmitter.emitRow(`row:delete`, appId, row)
+    ctx.eventEmitter &&
+      ctx.eventEmitter.emitRow(
+        `row:delete`,
```

Comment:
would probably be wroth changing this into an object specifier like:
```
{ appId: string, row: Row, table?: Table, oldRow?: Row, user: User }
```

This way user can be required and some elements can be optional, without needing to specify `null`.

Reply:
Done